external/googleapis/CMakeLists.txt

```diff
@@ -523,6 +526,7 @@ string(
            " google_cloud_cpp_api_client_protos"
            " google_cloud_cpp_api_field_behavior_protos"
            " google_cloud_cpp_api_http_protos"
+           " google_cloud_cpp_api_httpbody_protos"
```

Comment:
This list is used for backwards compatibility (see #5726).  We probably do not want to grow it.

Reply:
Removed.